hedera-node/src/main/java/com/hedera/services/fees/calculation/AutoRenewCalcs.java

```diff
@@ -137,7 +137,7 @@ long rbUsedBy(MerkleAccount account) {
 				.setCurrentKey(asKeyUnchecked(account.getAccountKey()))
 				.setCurrentlyHasProxy(account.getProxy() != null)
 				.setCurrentMemo(account.getMemo())
-				.setCurrentNumTokenRels(account.tokens().numAssociations())
+				.setCurrentNumTokenRels(account.getTokenAssociationMetadata().numAssociations())
```

Comment:
It's annoying, but this breaks the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter), which restricts which objects a method is allowed to call other methods on:

> In particular, an object should avoid invoking methods of an object returned by another method. For many modern object oriented languages that use a dot as field identifier, the law can be stated simply as "use only one dot". That is, the code `a.m().n()` breaks the law where `a.m()` does not.

So we'd prefer to still have a `MerkleAccount.numAssociations()` method. 😁